### PR TITLE
Backend: services endpoints

### PR DESCRIPTION
**Pull request by Pankai222**
Endpoints for retrieving each type of service

### CODE REVIEW

**Comment by AndreasPB** [comment]
It's probably easier to work with if it's split up since they are used in different parts of the component (ezpz #each of similar looking data for the phones)

File: backend/app/main.py [hunk 1]
```diff
@@ -18,6 +21,20 @@ async def running_total(purchase: Purchase):
...
 
+@app.get("/cellphones", response_model=list[CellPhone])
+async def get_cellphones():
+
+    services = [
+        CellPhone(name="Motorola G99", price=800, regularity=Regularity.ONCE),
+        CellPhone(name="iPhone 99", price=6000, regularity=Regularity.ONCE),
+        CellPhone(name="Samsung Galaxy 99", price=1000, regularity=Regularity.ONCE),
+        CellPhone(name="Sony Xperia 99", price=900, regularity=Regularity.ONCE),
+        CellPhone(name="Huawei 99", price=900, regularity=Regularity.ONCE),
```
Comment (AndreasPB) [inline backend/app/main.py:32]: You don't have to set the regularity, ONCE should be the default value

**Review by AndreasPB — COMMENTED** [review]
Looks nice so far - Are you gonna make the issue into this PR or make more PRs?

File: backend/app/main.py (hunk 1 above)
Comment (Pankai222) [reply backend/app/main.py:32]: Coding at 1:30 in the morning be like

**Comment by Pankai222** [comment]
Might as well have the whole issue completed in this PR. We need it for frontend ASAP

**Comment by AndreasPB** [comment]
> Might as well have the whole issue completed in this PR. We need it for frontend ASAP

Have you logged into your `hub`-cli tool up yet? ;D

**Comment by Pankai222** [comment]
Couldn't find the right command to reset credentials :/

**Review by AndreasPB — APPROVED** [review]
:shipit:

**Comment by AndreasPB** [comment]
Men rebase først

**Pull request merged**